packages/salesforcedx-vscode-core/src/commands/forceProjectCreate.ts

```diff
@@ -102,6 +118,43 @@ export interface ProjectName {
   projectName: string;
 }
 
+export interface ProjectTemplate {
+  projectTemplate: string;
+}
+
+export class SelectProjectTemplate
+  implements ParametersGatherer<ProjectTemplate> {
+  private readonly prefillValueProvider?: () => string;
+
+  constructor(prefillValueProvider?: () => string) {
+    this.prefillValueProvider = prefillValueProvider;
+  }
+
+  public async gather(): Promise<
+    CancelResponse | ContinueResponse<ProjectTemplate>
+  > {
+    const items: vscode.QuickPickItem[] = [
+      new ProjectTemplateItem(
+        projectTemplateEnum.standard,
+        'force_project_create_standard_template'
+      ),
+      new ProjectTemplateItem(
+        projectTemplateEnum.empty,
+        'force_project_create_empty_template'
+      ),
+      new ProjectTemplateItem(
+        projectTemplateEnum.analytics,
+        'force_project_create_analytics_template'
+      )
+    ];
+
+    const selection = await vscode.window.showQuickPick(items);
+    const projectTemplate = selection
+      ? selection.label
+      : projectTemplateEnum.standard;
+    return { type: 'CONTINUE', data: { projectTemplate } };
```

Comment:
This needs to include a `type : 'CANCEL'` or it will allow the user to continue executing the command and skip selecting a project template.